sev-types/src/lib.rs

```diff
@@ -16,9 +16,22 @@
 //! the SEV API specification document version; the specification publication
 //! date; the link to the SEV API specification.
 
-#![no_std]
 #![deny(missing_docs)]
 #![deny(clippy::all)]
+#![allow(missing_docs)]
 
 pub mod command;
 pub mod platform;
+
+#[repr(C)]
+#[derive(Copy, Clone, Debug, Default, PartialEq, Eq, PartialOrd, Ord)]
+pub struct Build {
```

Comment:
Since the struct represents the platform firmware build, I'd argue that it belongs in the `platform` module.

Furthermore, this struct will require a doc comment describing it.